src/runtime/runtime-test.js

```diff
@@ -1687,7 +1687,7 @@ describes.realWin('ConfiguredRuntime', {}, env => {
 
       const button = runtime.createButton();
       await button.click();
-      expect(count).to.equal(1);
+      expect(count).to.not.equal(0);
```

Comment:
I dont understand the reason for this change. Is this carried over from a previous change or something changed in how button clicks are counted?

Reply:
I mentioned this too, it's been fixed 👍